metadata/android/en-US/short_description.txt

```diff
@@ -1 +1 @@
-Lightweight and privacy-conscious virtual keyboard for Android.
+Bàn phím ảo gọn nhẹ và tôn trọng quyền riêng tư cho Android.
```

Comment:
This is the English description. You need to put that into a new file: `metadata/android/vi-VN/short_description.txt`.